src/ast.scm

```diff
@@ -274,6 +274,10 @@
 
 (define (quoted? e) (memq (car e) '(quote top core globalref outerref line break inert meta)))
 (define (quotify e) `',e)
+(define (unquote e)
+  (if (and (pair? e) (quoted? e))
```

Comment:
For the use case in this PR this should probably only remove `quote` and `inert`.